poetry.lock

```diff
@@ -1088,122 +1088,123 @@ test = ["coverage[toml]", "ddt (>=1.1.1,!=1.4.3)", "mock ; python_version < \"3.
 
 [[package]]
 name = "grimp"
-version = "3.8"
+version = "3.8.1"
```

Comment:
can skip: could you relock to resolve transitive h11 vulnerability (from today)?